src/main/java/net/neoforged/moddevgradle/internal/ModDevPlugin.java

```diff
@@ -259,13 +259,24 @@ public void apply(Project project) {
             Function<String, Provider<RegularFile>> jarPathFactory = suffix -> {
                 return minecraftArtifactsDir.zip(
                         // It's helpful to be able to differentiate the Vanilla jar and the NeoForge jar in classic multiloader setups.
-                        extension.getVersion().map(v -> "neoforge-" + v).orElse(extension.getNeoFormVersion().map(v -> "vanilla-" + v)),
+                        extension.getNeoForgeArtifact().map(art -> {
+                                    var split = art.split(":", 3);
+                                    return split[1] + "-" + split[2];
+                                })
+                                .orElse(extension.getNeoFormArtifact().map(v -> "vanilla-" + v.split(":", 3)[2])),
                         (dir, prefix) -> dir.file(prefix + "-minecraft" + suffix + ".jar"));
             };
             task.getCompiledArtifact().set(jarPathFactory.apply(""));
             task.getCompiledWithSourcesArtifact().set(jarPathFactory.apply("-merged"));
             task.getSourcesArtifact().set(jarPathFactory.apply("-sources"));
-            task.getResourcesArtifact().set(jarPathFactory.apply("-resources-aka-client-extra"));
+            task.getResourcesArtifact().set(minecraftArtifactsDir.zip(
+                    extension.getNeoForgeArtifact().map(art -> {
+                                var split = art.split(":", 3);
+                                return split[2] + "-" + split[1];
+                            })
+                            .orElse(extension.getNeoFormArtifact().map(v -> "vanilla-" + v.split(":", 3)[2])),
+                    (dir, prefix) -> dir.file("client-extra-aka-minecraft-resources-" + prefix + ".jar")
```

Comment:
Please comment that we name it this way because we cannot start with `forge-`.

Reply:
If it's not needed, revert this change?

Reply:
Yes, please check why `jarPathFactory.apply("-resources-aka-client-extra")` is not enough.